porcupine/plugins/filemanager.py

```diff
@@ -122,6 +125,24 @@ def rename(old_path: Path) -> None:
         tab.path = new_path / tab.path.relative_to(old_path)
 
 
+def paste(new_path: Path) -> None:
+    global copy_path
+    new_file_path = os.path.join(new_path, os.path.basename(copy_path))
+
+    if os.path.exists(new_file_path):
+        log.error("already exists - copy aborted")
+        return
```

Comment:
IMO it shouldn't be an error to copy from a directory into itself. The file should just get a different name, e.g. `foo.py` --> `foo (2).py` or something. Or we could ask the user how to name the file.

Reply:
Exactly, I wanted to ask you about it. I'm a Ubuntu user and the file explorer asks me how to call the new file when I add it to a folder which contains a file with the same name. What do you think is the right approach for Porcupine? I would be happy to implement it.

Reply:
I think it would be best to modify the renaming code so that the same dialog can be used to ask the new name when copying. If my copy was named `foo (2).py`, I would have to rename it myself anyway.